### ISSUE TITLE
Secure the credentials file

### ISSUE THREAD

**Issue by isaactzab** [okonek/tidal-cli-client#52]
First of all, Honor to @okonek, it's a great app.
My question is about if is possible improve more security in the credentials config file. Have the credentials exposed is insecure.

**Comment by okonek** [comment]
Yes, I know that the credentials management in my app is not very good, but I couldn't figure out any way of securing it. If you have an idea on how the password can be encrypted in this kind of open-source app, please tell me.
